herpetologica.csl

```diff
@@ -0,0 +1,139 @@
+    </group>
+  </macro>
+  <macro name="year-date">
+    <date variable="issued">
+      <date-part name="year"/>
+    </date>
+  </macro>
+  <macro name="edition">
+    <choose>
+      <if is-numeric="edition">
+        <group delimiter=" ">
+          <number variable="edition" form="ordinal"/>
+          <text term="edition" form="short"/>
+        </group>
+      </if>
+      <else>
+        <text variable="edition" suffix="."/>
+      </else>
+    </choose>
+  </macro>
+  <citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" collapse="year-suffix" cite-group-delimiter=",">
+    <layout prefix="(" suffix=")" delimiter="; ">
+      <group delimiter=" ">
+        <text macro="author-short"/>
+        <text macro="year-date"/>
+        <text variable="locator"/>
+      </group>
+    </layout>
+  </citation>
+  <bibliography hanging-indent="true" et-al-min="10" et-al-use-first="3" et-al-use-last="true">
+    <sort>
+      <key macro="author"/>
+      <key variable="issued"/>
+    </sort>
+    <layout>
+      <text macro="author" suffix="."/>
+      <date variable="issued" prefix=" " suffix=".">
+        <date-part name="year"/>
+      </date>
+      <choose>
+        <if type="bill book graphic legal_case legislation motion_picture report song thesis" match="any">
+          <group delimiter=". " prefix=" "  suffix=".">
+            <text macro="title"/>
+            <text variable="volume"/>
+            <text macro="edition"/>
+            <text macro="editor"/>
+            <text macro="publisher"/>
+            <text variable="number-of-pages" suffix=" pp"/>
+          </group>
+        </if>
+        <else-if type="chapter paper-conference" match="any">
+          <group delimiter=". " prefix=" "  suffix=".">
+            <text macro="title"/>
+            <group delimiter=" ">
+              <text term="page"  plural="true" form="short" text-case="capitalize-first"/>
+              <text variable="page"/>
+              <text term="in"/>
+              <text variable="container-title"/>
+              <text macro="editor"/>
+            </group>
+            <text macro="publisher"/>
+          </group>
+        </else-if>
+        <else-if type="webpage">
+          <group delimiter=". " prefix=" "  suffix=".">
+            <text macro="title"/>
+            <text variable="container-title"/>
+            <group delimiter=" ">
+              <text term="available at" text-case="capitalize-first"/>
+              <text variable="URL"/>
+            </group>
+          </group>
+        </else-if>
+        <else>
+          <group delimiter=" " prefix=" ">
+            <text macro="title" suffix="."/>
+            <text variable="container-title" form="short"/>
+            <group delimiter=":" suffix=".">
+              <text variable="volume"/>
+              <text variable="page"/>
+            </group>
+            <text prefix="DOI: http://dx.doi.org/" variable="DOI"/>
```

Comment:
We use `https://doi.org/`, following CrossRef recommendation, even where the styleguide says otherwise. The dx format work, but is outdated.